### ISSUE TITLE
Rounding up average and categorizing as higher monthly fixed price

### ISSUE THREAD

**Issue by atus** [sindrebroch/ha-elvia#19]
## Version of the custom_component
0.3.0

## Configuration

```yaml

Add your logs here.

```

## Describe the bug
The integration is claiming the user is in the `10-15 kWh/h` category even though it shows the average is <10kWh/h and the three current max hours confirm the calculation.

![image](https://github.com/sindrebroch/ha-elvia/assets/1918329/08766d67-044a-445c-badb-973812495265)

Perhaps there is some sort of rounding that pushes everyone with 9.5+ kWh/h average onto the higher bucket?

## Debug log

```text

Add your logs here.

```

**Comment by sindrebroch** [comment]
Hi. Does the data not match the Elvia-app or website? The integration itself is not doing any calculations, it is just fetching data from the elvia-API and displaying it in sensors.

**Comment by atus** [comment]
No, it doesn't match. Here's a current snapshot. ![image](https://github.com/sindrebroch/ha-elvia/assets/1918329/f959e089-0de5-4cbc-890a-4886e2af932f)

**Comment by sindrebroch** [comment]
Weird. The integrations still doesn't do any calculations on its own, so it should just return the value from the API.

Maybe it hasn't refreshed its value since last month? Where you at 10-15kWh last month?

**Comment by atus** [comment]
No. Nor the month before that ![image](https://github.com/sindrebroch/ha-elvia/assets/1918329/33cdaf0c-e5d7-4887-ab0c-8fc7241bcd21)

**Comment by sindrebroch** [comment]
There are a couple of ways we could debug this. I would need to see the response from the API.

1. You could turn on debugging on the integration-page, reload the integration, then find the logs with the response and send it to me. If you can email them to me at sindrebroch@gmail.com if you don't want to post it here. Not sure how sensitive the response is. 
http://<your-homeassistant-ip>/config/integrations/integration/elvia

2. You could log into the Elvia developer portal, query the GridTariffApi > meteringspointsgridtariffs

![image](https://github.com/sindrebroch/ha-elvia/assets/10772085/fe2e2b58-1966-4c11-bd78-70237b98c92e)

https://elvia.portal.azure-api.net/docs/services/gridtariffapi/operations/post-digin-api-v-tariffquery-meteringpointsgridtariffs/console

**Comment by atus** [comment]
This is for my meter
```
"meteringPointsAndPriceLevels": [
    {
        "currentFixedPriceLevel": {
            "id": "d4087fd1-bd1b-4765-ae70-5aad06c6e4de",
            "levelId": "4936a5c4-7326-4a04-9614-d89a4463c76e"
        },
        "meteringPoints": [
            {
                "meteringPointId": "xxx",
                "levelValue": null,
                "lastUpdated": "2023-12-17T16:00:00+01:00"
            }
        ]
    }
]
```

This is the relevant price. Which is what you show.
```
{
    "id": "4936a5c4-7326-4a04-9614-d89a4463c76e",
    "valueMin": 10.0001,
    "valueMax": 15.0001,
    "nextIdDown": "b1c52896-35c6-4320-ad10-f6324b541bdf",
    "nextIdUp": "2a965304-7f12-47ec-9343-bf4777741c99",
    "valueUnitOfMeasure": "kWh/h",
    "monthlyTotal": 420,
    "monthlyTotalExVat": 336,
    "monthlyExTaxes": 336,
    "monthlyTaxes": 84,
    "monthlyUnitOfMeasure": "kr/month",
    "hourPrices": [
        {
            "id": "f300d806-7ccc-47cf-b595-b783650f94df",
            "numberOfDaysInMonth": 31,
            "total": 0.5645,
            "totalExVat": 0.4516
        }
    ],
    "levelInfo": "Power consumption: 10-15 kWh/h",
    "currency": "NOK",
    "monetaryUnitOfMeasure": "kr/hour"
},
```
I am still not in the in the higher tier as this API response implies.

**Comment by atus** [comment]
I've dug around their APIs and can't make any sense of this. My average is <10.0001 but this API returns me in the 10-15 category. The app still returns 5-10 category

Time to ask Elvia about this?

**Comment by sindrebroch** [comment]
Weird. Yes I think talking to Elvia is a good idea. 

Good to know that the integrations reflects the API correctly at least 👍
